Src/Sentry.Xamarin/SentryXamarinOptions.cs

```diff
@@ -8,6 +8,17 @@ namespace Sentry
     /// </summary>
     public class SentryXamarinOptions : SentryOptions
     {
+        /// <summary>
+        /// Attaches screenshots from the app to events automatically whenever possible.
+        /// </summary>
+        public bool AttachScreenshots { get; set; }
```

Comment:
This should probably include a note about PII. 
There could be potentially private information in the screenshot. For example in bank or chat apps.

Reply:
I agree